{"name": "Не успешая авторизация", "status": "passed", "steps": [{"name": "Нажать Аккаунт", "status": "passed", "start": 1684320351245, "stop": 1684320351292}, {"name": "Нажать Войти", "status": "passed", "start": 1684320351292, "stop": 1684320351440}, {"name": "Войти как 'qwerty@qwerty.qwerty' 'VTer38CX'", "status": "passed", "start": 1684320354442, "stop": 1684320354652}, {"name": "Проверить что ошибка логина отображается", "status": "passed", "start": 1684320354653, "stop": 1684320355302}], "start": 1684320351245, "stop": 1684320355302, "uuid": "79252157-520b-4f2e-9da9-297134885690", "historyId": "8eaf6323d358d700d14848100a31cbc2", "testCaseId": "7b6113008a0e4d8a5bad2235a270375d", "fullName": "tests.test_login.TestLogin#test_login_fail", "labels": [{"name": "feature", "value": "Тесты логина"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-VLT8PU4"}, {"name": "thread", "value": "12164-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}